packages/astro/test/astro-doctype.test.js

```diff
@@ -24,7 +24,7 @@ describe('Doctype', () => {
 		expect(html).to.match(/^<!DOCTYPE html>/i);
 	});
 
-	it('Preserves user provided doctype', async () => {
+	it.skip('Preserves user provided doctype', async () => {
```

Comment:
meant to be skipped? If so, leave a comment so that future people don't think its a mistake. Otherwise, should be safe to remove.